spec/import_spec.rb

```diff
@@ -114,6 +114,37 @@ def upload_file!(name, ext='csv')
 
   end
 
+  context "authors already exist" do
+    before do
+      Author.create!(id: 1, name: "Jane", last_name: "Roe")
+      Author.create!(id: 2, name: "John", last_name: "Doe")
+    end
+
+    context "having authors with the same Id" do
+      before do
+        add_author_resource({
+          before_batch_import: ->(importer) do
+            Author.where(id: importer.values_at("id")).delete_all
+          end
+        })
+        visit '/admin/authors/import'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.